main.cpp

```diff
@@ -1,19 +1,88 @@
 #include <iostream>
 #include <cstdlib>
 
-int getNumber( int min, int count )
-{
-    return min + rand() % count ;
+int getNumber(int min, int count ) {
+    return min + rand() % count; 
 }
 
-int main()
-{
-    srand( time( nullptr ) );
+int largest(int arr[], int size) {
+    int max = arr[0];
+    for(int a = 0; a < size; a++) {
+        if(arr[a] > max) {
+            max = arr[a];
+        }
+    }
+    return max;
+}
+
+int smallest(int arr[], int size) {
+    int min = arr[0];
+    for(int b = 0; b < size; b++) {
+        if(arr[b] < min) {
+            min = arr[b];
+        }
+    }
+    return min;
+}
+
+int main() {
+
+    int diceThrows[100];
+    int diceSides[6]{};
+    int rarest{};
+    int commonest{};
+
+    srand(time(nullptr));
+    
+    for(int i = 0; i < 100; i++) {
+        diceThrows[i] = getNumber(1, 6);
+        switch (diceThrows[i])
+        {
+        case 1:
+            diceSides[0]++;
+            break;
+        case 2:
+            diceSides[1]++;
+            break;
+        case 3:
+            diceSides[2]++;
+            break;
+        case 4:
+            diceSides[3]++;
+            break;
+        case 5:
+            diceSides[4]++;
+            break;
```

Comment:
Можно привязатся к текущему индексу, 
```suggestion
        case 1:
        case 2:
        case 3:
        case 4:
        case 5:
            diceSides[i]++;
            break;
```
Нету обработки 6-й стороны

Reply:
6-та сторона обробляється в дефолт кейсі

Reply:
Ваш метод не працює як слід. Кожен елемент інкрементується по порядку (навіть не залежно від сторони яка випала) один раз і далі кількість випадінь кожної сторони залишається сталою

Reply:
1) Обрабатывать в дефолтном кейсе какой-то вариант - это некорректно. Default для ситуаций, когда происходит исключительная ситуация.
2) Я не буду настаивать не изменении кода, но пересмотрите, тот вариант, который я предложил - рабочий. Единственное, не досмотрел по индексу - там i + 1

Reply:
Не забудьте проверить выпавший номер и количество элементов массива.

Reply:
додав case 6 та переписав дефолт